includes/ajax.php

```diff
@@ -476,10 +478,16 @@ function tsml_ajax_import() {
 			}
 			
 			//add custom meeting fields if available
-			foreach (array('types', 'group', 'data_source', 'venmo', 'conference_url', 'conference_phone') as $key) {
+			foreach (array('types', 'data_source', 'conference_url', 'conference_phone') as $key) {
```

Comment:
this change should come out; `group` and `venmo` are associated with the group, but this is attaching custom fields to the meeting